### ISSUE TITLE
Nextcloud app android does not recognize active e2e application

### ISSUE THREAD

**Issue by tigernero79** [nextcloud/android#11155]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

I just upgraded Nextcloud from version e 25.0.1 to version 25.0.2 and activated the Nextcloud e2e plugin.  via web everything works fine but if you try to access an encrypted folder e2e android application tells me that encryption is disabled on the server side, which is not true as it is also active and running via Nextcloud desktop

### Expected behaviour

I just upgraded Nextcloud from version e 25.0.1 to version 25.0.2 and activated the Nextcloud e2e plugin.  via web everything works fine but if you try to access an encrypted folder e2e android application tells me that encryption is disabled on the server side, which is not true as it is also active and running via Nextcloud desktop

### Actual behaviour

I just upgraded Nextcloud from version e 25.0.1 to version 25.0.2 and activated the Nextcloud e2e plugin.  via web everything works fine but if you try to access an encrypted folder e2e android application tells me that encryption is disabled on the server side, which is not true as it is also active and running via Nextcloud desktop

![Screenshot_20221208-182122](https://user-images.githubusercontent.com/35575744/206525862-c727a4ea-0b87-44d3-b259-5999aaa291af.png)

### Android version

12

### Device brand and model

Pixel 6 pro

### Stock or custom OS?

Stock

### Nextcloud android app version

3.22.3

### Nextcloud server version

25.0.2

### Using a reverse proxy?

I don't know

**Comment by tigernero79** [comment]
no, but creating another e2e folder in the end it asked me for the 12 words and I entered I had to remove data in old e2e folders and put it back because it wouldn't let me download them giving me server error.

**Comment by tigernero79** [comment]
unfortunately even on a new nextcloud server 25.0.2 installation everything works fine with E2E but when I configure the instance on nextcloud the android application sees the encrypted folder padlock symbol but if I click to enter and memorize the 12-word passphrase for the first time it always says encryption it is disabled on the server side, which is not true

**Comment by tigernero79** [comment]
operations I perform on a clean install of Nextcloud v.25.0.2.  active e2e plugin.  start Nextcloud android app e2e encryption is not active, open Nextcloud desktop application I can encrypt the folder from PC.  return to Nextcloud android application sees folder with padlock but does not ask for passphrase it tells me server side encryption does not arrive, which is not true because in truth it is.

**Comment by joshtrichards** [comment]
> server side encryption does not arrive, which is not true because in truth it is.

I thought you were using E2EE (which is client side encryption not server side). Can you confirm server side encryption isn't configured simultaneously?

**Comment by tigernero79** [comment]
@joshtrichards 

I confirm that server-side encryption is disabled, as I confirm that the Android application does not allow you to delete files in e2ee folders, there is no entry in the delete menu on NextCloud 27.1.2 even while on the iOS application everything works.
